packages/@orchidui-vue/package.json

```diff
@@ -24,6 +24,18 @@
     ".": {
       "import": "./dist/index.js"
     },
+    "./TextEditor": {
+      "import": "./dist/TextEditor.js"
+    },
+    "./ComplexCalendar": {
+      "import": "./dist/ComplexCalendar.js"
+    },
+    "./ComplexDatePicker": {
+      "import": "./dist/ComplexDatePicker.js"
+    },
+    "./SingleFileUpload": {
+      "import": "./dist/SingleFileUpload.js"
+    },
```

Comment:
Why we could not hold same structure with these components (I mean export file in folder with component)?